api-reference/introduction.mdx

```diff
@@ -9,36 +9,41 @@ description: 'Comprehensive guide to the Ace AI API endpoints'
 
 ## Welcome to the Ace API
 
-The Ace AI API allows developers to interact with AI agents and related services to fine-tune models, generate synthetic data, clean datasets, and more. This API is designed to handle client requests efficiently and communicate with backend services via Kafka for scalable and decoupled processing.
+The Ace AI API allows developers to interact with AI-related services to:
+- Generate synthetic data.
+- Upload and download datasets using signed URLs.
+- Evaluate datasets with custom AI prompts.
+- (Planned) Perform data refinement and other advanced data tasks.
 
-With this API, you can:
-- Fine-tune AI agents using datasets.
-- Request agents to generate synthetic data.
-- Access and manage datasets with signed URLs.
-- Leverage AI capabilities for custom tasks.
+These services are designed to handle client requests efficiently. Additionally, the API can communicate with backend services (including Kafka) for scalable and decoupled processing of tasks like data generation and model training.
```

Comment:
Also add the following: 

"All data will be subject to scrutiny from our team of domain-expert human evaluators."